app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,39 @@
+<resources>
+    <string name="app_name">Plantare</string>
+
+    <string name="title_section1">Mapa Plantas</string>
```

Comment:
my recommendation is to change "title_section1" by "plants_map" or "title_section_plants_map"